libconnman-qt/networkmanager.h

```diff
@@ -58,6 +58,8 @@ class NetworkManager : public QObject
     Q_PROPERTY(QString GpsTechnology READ gpsTechnologyPath CONSTANT)
     Q_PROPERTY(QString EthernetTechnology READ ethernetTechnologyPath CONSTANT)
 
+    Q_PROPERTY(QVariantMap TetheringClients READ getTetheringClients NOTIFY tetheringClientsChanged)
```

Comment:
QVariantMap is not a nice API. Maybe it's a start but at minimum I'd reserve the right to switch the type into something better. I.e. somehow marked experimental or warnings like that.

For another thing it feels peculiar having this in the NetworkManager. The rest of the tethering APIs are now in NetworkTechnology, and tethering can be started for wlan and bluetooth. How should one know which technology a client is using?

Reply:
You seem to have misread that this is a WIP.

Read the ConnMan side before commenting on where to put what. I'm not going to even bother to comment this more.

Reply:
Since you've made this a public PR, I assumed you're willing or even hoping to get some feedback. But sure if you want I can also come back after wip removal and tell how this should be re-done.

Reply:
Well with that attitude of yours, I think `/dev/null` is correct place for your comments. Do you go to say  to a painter that YOU want him to paint your way?

I make it a WIP so I can distance myself from own work to see it better in a different light. If you wish to rant on unfinished work that is still evolving, fine. But those comments are ignored, hence the plan.